quadratic_equation.py

```diff
@@ -0,0 +1,54 @@
+import math
+
+def get_numbers_input() -> list[str]:
+    """
+    getting from terminal data and returning it
+    :return: List of three strings separated by spaces
+    """
+    return input("Enter list of three digits by space: ").split()

```

Comment:
only three coefs, no needs to append anything, looks like tuple is better option here then list

Reply:
done